### ISSUE TITLE
Swimming pool. Evaporation Heat Loss and Pool Cover Schedule issue

### ISSUE THREAD

**Issue by zhukovgreen** [NREL/EnergyPlus#4852]
I noticed that if the room dew-point temperature is rising the evaporation heat loss from the swimming pool is also rising. I assume it should be exactly opposite behavior, because delta (Pw–Pdp) is reducing.

![image](https://cloud.githubusercontent.com/assets/10520003/7136934/aa6f0c9c-e2b5-11e4-8e5d-f23c565d6d33.png)
![image](https://cloud.githubusercontent.com/assets/10520003/7137063/bb9e2ff6-e2b6-11e4-9bfc-3be29757314d.png)

**Comment by ecoeficiente** [comment]
I see it the other way round. When the pool is covered (no evaporation) the dew point reduces. With evaporation, the dewpoint rises. That is the supposed behaviour.

**Comment by zhukovgreen** [comment]
I was tracking an evaporation as a function of room air dew point (cover was always at 0%). In this case once the air become more saturated with water vapor the evaporation rate should start going down, but we have opposite behave.

**Comment by zhukovgreen** [comment]
![image](https://cloud.githubusercontent.com/assets/10520003/7138143/a80eac2e-e2be-11e4-8329-3be1a32298bc.png)

**Comment by rraustad** [comment]
// Evaporation calculation:
// Evaporation Rate (lb/h) = 0.1 \* Area (ft2) \* Activity Factor \* (Psat,pool - Ppar,air) (in Hg)
// So evaporation rate, area, and pressures have to be converted to standard E+ units (kg/s, m2, and Pa, respectively)
// Evaporation Rate per Area = Evaporation Rate \* Heat of Vaporization / Area of Surface
PSatPool = PsyPsatFnTemp( Pool( PoolNum ).PoolWaterTemp, RoutineName );

PParAir = PsyPsatFnTemp( MAT( ZoneNum ), RoutineName ) \* PsyRhFnTdbWPb( MAT( ZoneNum ), ZoneAirHumRatAvg( ZoneNum ), OutBaroPress );

if ( PSatPool < PParAir ) PSatPool = PParAir;

EvapRate = ( 0.1 \* ( Surface( SurfNum ).Area / CFA ) \* Pool( PoolNum ).CurActivityFactor \* ( ( PSatPool - PParAir ) \* CFinHg ) ) \* CFMF \* Pool( PoolNum ).CurCoverEvapFac;

Pool( PoolNum ).MakeUpWaterMassFlowRate = EvapRate;

EvapEnergyLossPerArea = -EvapRate \*  PsyHfgAirFnWTdb( ZoneAirHumRatAvg( ZoneNum ), MAT( ZoneNum ) ) / Surface( SurfNum ).Area;

Pool( PoolNum ).EvapHeatLossRate = EvapEnergyLossPerArea \* Surface( SurfNum ).Area;

**Comment by rraustad** [comment]
1) why can't the moisture in air migrate to the pool?
if ( PSatPool < PParAir ) PSatPool = PParAir;

2) Might EvapEnergyLossPerArea be a function of Tpool and Wsat,pool ?

**Comment by zhukovgreen** [comment]
@rraustad I think that considering the application good practice that the water temperature should be less @~2C than air, such situation would never occur

**Comment by mjwitte** [comment]
Helpdesk ticket 11450, user says that the pool cover schedule is not working.

> "Indoor Pool Evaporative Heat Loss Rate [W]" report variable does not change when you partially cover the pool. That is if you want the pool cover on during the night and off during day. The "Indoor Pool Evaporative Heat Loss Rate [W]" report variable only change when the pool is either covered from 24 hrs or uncovered for 24 hrs ( on or off). I have attached the example Energy Plus file instead of my own simulation IDF so that we eliminate the risk that my own IDF has incorrect inputs.

[5ZoneSwimmingPool test.idf.txt](https://github.com/NREL/EnergyPlus/files/342116/5ZoneSwimmingPool.test.idf.txt)

**Comment by RKStrand** [comment]
@mjwitte Thanks for the extra info and the input file, Mike.  I will add it to my list of bugs to look at and also put it in Pivotal Tracker.

**Comment by RKStrand** [comment]
@ZhukovGreen I hope to be looking at this issue very soon.  I am interested to see what the actual zone air temperature is for the data that you posted.  You said that the dew-point temperature is changing, but in order to see just how that is changing we need to know how humidity ratio not relative humidity is varying. Have you posted your input file already (or is it the same file that @mjwitte posted)?

**Comment by mjwitte** [comment]
@RKStrand 5ZoneSwimmingPool test.idf (attached to this issue [above](https://github.com/NREL/EnergyPlus/issues/4852#issuecomment-229748983) is from a different user.

There is a different file in \EnergyPlusDevSupport\DefectFiles\4000s\4852\ #ESS0712_SwPoolApp.idf which I believe is the file from @ZhukovGreen

**Comment by zhukovgreen** [comment]
Hello @RKStrand,

yes it is correct. The file #ESS0712_SwPoolApp.idf  is those one which was used in my test

**Comment by RKStrand** [comment]
@ZhukovGreen Just wanted to let you know that I did identify a bug in the code that was causing problems with the calculation.  There is a fix and this will hopefully be corrected in the next version of EnergyPlus.

@mjwitte I have looked at the issue for Helpdesk ticket 11450 that was noted above.  I do not see a problem with the cover being on/off for partial days in the test suite file.  I have, however, added new report variables and made slight changes to the test suite input file (5ZoneSwimmingPool.idf) to make things a bit clearer and more realistic.  The new output variables will also make both of these issues easier to diagnose.  If the user who generated the Helpdesk ticket wants to send in an input file for further review, that is fine.  However, I do not see the trend that the user noted in the branch that I am working in.

**Comment by zhukovgreen** [comment]
@RKstrand great, thank you.

**Comment by RKStrand** [comment]
@mjwitte Looking at this again, trying to figure out if there was anything in the docs that might be confusing and perhaps causing a user to not understand how evaporation rate might vary or not since I am not seeing what the issue the helpdesk question user is.  Having done that, I am seeing some mistakes in the engineering reference that must have been introduced with the various changes in documentation format since looking back to the NFP document there is clearly a problem.  There are some missing minus signs, a power that wasn't properly converted, and an entire table that was replaced with a different table.  I'm going to clean this up as well as part of this and also add a few things to clarify the effect of the pool cover on evaporation.

**Comment by zhukovgreen** [comment]
@RKStrand Hello Rick,
Am I understand the problem correctly if we have inside the room +30C, RH70 then we some evaporation rate A. if the partial water pressure is rising (for instance +30C, RH75), then the evaporation rate B should be smaller than A. Infact my screen shot shows opposite behaviour

**Comment by RKStrand** [comment]
@ZhukovGreen Yes, you are understanding the problem correctly.  As the partial water pressure rises, then the evaporation rate should be dropping based on what the model is supposed to be doing.  In looking at your file, the hourly data didn't really reveal a distinct pattern.  It wasn't until running your file through the debugger, increasing the frequency of the output to "detailed", and adding two new output variables that I was able to determine that the evaporation rate was actually being affected by the number of system time steps versus zone time steps.  Every time the system time step shortened the evaporation rate was changing.  With the new output variables (partial pressure of water vapor and saturation pressure at the pool water temperature), it was revealed that the partial pressure of water vapor was not correct--that in fact is wasn't using the correct variable inside the code.  Using your input file and the corrected code, I was able to verify the proper trend in the output and it is definitely different than the data that you reported above.  So, this was definitely a mistake in the code and I apologize for any problems or inconvenience that it might have caused you.  Thank you for pointing this out as your understanding of the issue was correct and the code had a problem that needed to be fixed.
